### ISSUE TITLE
Grace notes

### ISSUE THREAD

**Issue by user146801** [musescore/MuseScore#20865]
### Issue type

Choose option...

### Bug description

When you put grace notes before and after a note, it doesn't play the sound of both of them.

### Steps to reproduce

1.place note
2.put grace notes before
3.put grace notes after
4.click play
5.It plays only the sound of the grace notes you put first (in this case before notes)

### Screenshots/Screen recordings

https://musescore.org/sites/musescore.org/files/2024-01/Hungroise.mscz
https://musescore.org/sites/musescore.org/files/2024-01/Captura%20de%20pantalla%202024-01-05%20222926.png

### MuseScore Version

4.2.0

### Regression

I don't know

### Operating system

Windows 11

**Comment by wizofaus** [comment]
Are there existing issues around how configurable playback of grace notes should be? i.e. whether they should start before the beat, how fast to play them etc.?

**Comment by MarcSabatella** [comment]
Yes, for example https://github.com/musescore/MuseScore/issues/20494 and https://github.com/musescore/MuseScore/issues/20885
